go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/brancz/kube-rbac-proxy
 
-go 1.24.2
+go 1.24.6
```

Comment:
This has no effect on the go build. This one only indicates which minimal Go version is required. The actual one is in the github actions.

Reply:
I see, thanks for checking
IIUC there is nothing that needs to be updated [here](https://github.com/brancz/kube-rbac-proxy/blob/master/.github/workflows/build.yml#L5) because it has already the minor version set so I suppose it will pick the latest available patch version of that minor at the time of the image build, right?